README.md

```diff
@@ -88,15 +88,16 @@ Game Boy (Advance), Nintendo DS ROMs and Nintendo 3DS FIRMs.
 
 ## Scripting
 
-We provide a way to interact with Rizin from Python/Haskell/OCaml languages
+We provide a way to interact with Rizin from Python/Haskell/OCaml/Ruby languages
```

Comment:
Missing Go and Rust then.

Reply:
Sure. Then, we'll use the comma instead of the forward slash?

Reply:
Yes, it's probably better.